pages/components/attention.html

```diff
@@ -87,7 +87,7 @@ <h3 class="mt-24 mb-16">Popover</h3>
           <w-attention placement="bottom" popover>
             <button
               id="target"
-              class="group block relative break-words last-child:mb-0 p-16 rounded-8 bg-aqua-50"
+              class="group block relative break-words last:mb-0 p-16 rounded-8 bg-aqua-50"
```

Comment:
Do you know if there is a reason for the `group` class here? 🤔

Reply:
haha yeah, I dunno. that's old stuff 🤔
we should clean up these docs later as we don't use them as docs docs😄